Fixed #34577 -- Added escapeseq template filter. (#16877)
django/django: docs/releases/5.0.txt

Templates
~~~~~~~~~

> * The new :tfilter:`escapeseq` template filter allows escaping list items, 
>   marking each item in the list as a string that should be auto-escaped.

Reviewer: ```suggestion
* The new :tfilter:`escapeseq` template filter applies :tfilter:`escape` to
  each item in a sequence.
```
A couple of nitpicks 😊
 - Avoid repetition (ie don't say x escapes list items, marking each item to be escaped)
 - avoid specific types (say sequence)
 - auto-escape -> escape? 🤔

Templates
~~~~~~~~~

> * There's a new :tfilter:`escapeseq` template filter which works just like

Reviewer 2: ```suggestion
* The new :tfilter:`escapeseq` template filter allows ...
```
